app/views/laa_references/_form.html.haml

```diff
@@ -1,11 +1,14 @@
-= form_with(url: '/laa_references', local: true) do |f|
+= form_with(model: @link_attempt, url: '/laa_references', local: true) do |f|
+  - if @link_attempt
+    =f.govuk_error_summary
 
   = render 'defendants/defendant'
 
   = render 'defendants/offences'
 
   = f.hidden_field(:id, value: params[:id])
   = f.hidden_field(:defendant_id, value: @defendant.id)
+  = f.hidden_field(:defendant_asn, value: @defendant_asn)
```

Comment:
ideally would not need this as we have an id from which we can get a defendant, and hence an asn or nino